example-get-started/code/src/requirements.txt

```diff
@@ -1,3 +1,4 @@
+dvclive
```

Comment:
I'd recommend to specify the version upper limit :)

Reply:
Indeed! Need to release a version first 😄